recipes/pyscenic/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "pyscenic" %}
+{% set version = "0.12.1" %}
+{% set sha256 = "ae8fafa707d2578ffe08f9eed85f14a4cd9e1b53d57217420e2e956f0a8ddba2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  - url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+    sha256: "{{ sha256 }}"
+    folder: pyscenic-0.12.1
+
+  - url: "https://github.com/aertslab/ctxcore/archive/refs/tags/0.2.0.tar.gz"
+    sha256: "a7ebf0f2625641b76a390993e12042e92fff7d0ac242c7fad5e3bff3ff3cd67a"
+    folder: ctxcore-0.2.0
+  - url: "https://files.pythonhosted.org/packages/source/m/multiprocessing_on_dill/multiprocessing_on_dill-3.5.0a4.tar.gz"

```

Comment:
Working out a license like that could take a long time, in the meantime this could be used like this and possibly improved once that is resolved. There are many packages in bioconda that recourse to pip installing dependencies that are not available in conda, for instance https://github.com/bioconda/bioconda-recipes/blob/def3a07b68330abb40f7650b3747b8bee189ab66/recipes/latch/build.sh

Reply:
Thanks, I see your point. I will get some clarity on this from the core team.